docs/documentation/components/tabs.html

```diff
@@ -16,60 +16,78 @@
 
 {% capture tabs_example %}
 <div class="tabs">
-  <ul>
-    <li class="is-active"><a>Pictures</a></li>
-    <li><a>Music</a></li>
-    <li><a>Videos</a></li>
-    <li><a>Documents</a></li>
+  <ul role="tab">
+    <li role="presentation" class="is-active">
+      <a href="#picture-panel" id="picture-tab"
+         role="tab" aria-controls="picture-panel" aria-selected="true">Pictures</a>
+    </li>
+    <li role="presentation">
+      <a href="#music-panel" id="music-tab"
+         role="tab" aria-controls="music-panel" aria-selected="false">Music</a>
+    </li>
+    <li role="presentation">
+      <a href="#video-panel" id="video-tab"
+         role="tab" aria-controls="video-panel" aria-selected="false">Videos</a>
+    </li>
+    <li role="presentation">
+      <a href="#docs-panel" id="docs-tab"
+         role="tab" aria-controls="docs-panel" aria-selected="false">Documents</a>
+    </li>
   </ul>
 </div>
+
+<section id="picture-panel" role="tabpanel" tabindex="-1" aria-labelledby="picture-tab">Pictures Content</section>
+<section id="music-panel" role="tabpanel" tabindex="-1" aria-labelledby="music-tab" class="is-hidden">Music Content</section>
```

Comment:
to make the components work well if there's no JS we need to append the `is-hidden` class by JS. not in plain HTML.